Index: core/mui/gtk_extension.rb
===================================================================
--- core/mui/gtk_extension.rb	(revision 415)
+++ core/mui/gtk_extension.rb	(working copy)
@@ -185,6 +185,8 @@
   else
     if command_exist?('xdg-open')
       command = 'xdg-open'
+    elsif command_exist?('open')
+      command = 'open'
     else
       command = '/etc/alternatives/x-www-browser' end
     bg_system(command, url) end end
